itou/prescribers/management/commands/update_refused_prescriber_organizations_kind.py

```diff
@@ -0,0 +1,60 @@
+from django.core.management.base import BaseCommand
+from django.db.models import Count
+
+from itou.prescribers.models import PrescriberOrganization
+
+
+class Command(BaseCommand):
+    """
+    The command updates "kind" value of PrescriberOrganizations with "refused" authorization status
+    """
+
+    help = "Update kind of prescriber_organization with REFUSED authorization status"
+
+    def show_counts(self, prescriber_orgs_refused):
+
+        self.stdout.write("Number of Prescriber Organizations with REFUSED authorization status :")
+
+        prescriber_orgs_count = prescriber_orgs_refused.values("kind").annotate(Count("kind"))
+
+        for p in prescriber_orgs_count:
+            self.stdout.write(f"authorization_status=REFUSED prescriber_kind={p['kind']} count={p['kind__count']}")
+
+        self.stdout.write(
+            f"authorization_status=REFUSED prescriber_kind=TOTAL count={prescriber_orgs_refused.count()}"
+        )
+
+    def handle(self, **options):
+
+        # collect prescribers with refused status
+        prescriber_orgs_refused = PrescriberOrganization.objects.filter(
+            authorization_status=PrescriberOrganization.AuthorizationStatus.REFUSED
+        )
+        self.show_counts(prescriber_orgs_refused)
+
+        # collect all prescribers with siret in prescriber_orgs_refused to exclude them
```

Comment:
```suggestion
        # Collect all prescribers with siret in prescriber_orgs_refused to exclude them.
```

Une majuscule et un point siouplé, sinon je passe à la ligne pour lire la suite et je me rends compte qu'il s'agit d'une nouvelle phrase et mon cerveau grille.